testing/testdata/wasm_util.go

```diff
@@ -1,14 +1,21 @@
-package testapp
+package testdata
 
 import (
 	"os"
 	"os/exec"
 
+	owasm "github.com/bandprotocol/go-owasm/api"
+
 	"github.com/bandprotocol/chain/v2/x/oracle/types"
 )
 
-func compile(code []byte) []byte {
-	compiled, err := OwasmVM.Compile(code, types.MaxCompiledWasmCodeSize)
+func Compile(code []byte) []byte {
+	owasmVM, err := owasm.NewVm(10)
+	if err != nil {
+		panic(err)
+	}
+
+	compiled, err := owasmVM.Compile(code, types.MaxCompiledWasmCodeSize)
```

Comment:
is the Owasm in test_helpers is still in use

Reply:
used in line 549